spring-cloud-dataflow-app-launcher/src/main/java/org/springframework/cloud/dataflow/app/resolver/MavenProperties.java

```diff
@@ -198,6 +218,12 @@ public void setPassword(String password) {
 					this.getRemoteRepositories()));
 		}
 		properties.put(MAVEN_PREFIX + "offline", String.valueOf(this.isOffline()));
+		if (this.getConnectTimeout() > 0) {
```

Comment:
this means we would not pass thru a value of 0, which is valid... but maybe that's ok since that means NO timeout (although a user could pass any negative value to accomplish the same).... another option is to use `Integer` instead of `int` and check for `null` as the indicator to rely on the system's own default